src/torch/README.md

```diff
@@ -0,0 +1,113 @@
+# SIRF-PyTorch Wrapper
+This wrapper provides a bridge between the [SIRF](https://github.com/SyneRBI/SIRF) (Synergistic Image Reconstruction Framework) library and PyTorch, enabling the use of SIRF's image reconstruction operators and objective functions within PyTorch's automatic differentiation (autodiff) framework. 
+
+## Forward and backward clarification
+
+The use of the terms forward and backward have different meaning given the context:
+* `torch.autograd.Function`: the `forward` method (forward pass) is the evaluation of the function, and `backward` method (backward pass) computes the (scaled directional) derivative, i.e. the Vector-Jacobian-adjoint-Product (VJP), from output to input.
```

Comment:
Why do you refer to the derivatives as "scaled"?

Reply:
The vector isn't neccessarily a unit vector. That said, I think I will just say "gradient" as that is correct and less confusing, maybe it will be more obvious this is just the chain rule. Also I say " Vector-Jacobian-adjoint-Product" I need to take the adjoint out... I will work on this a bit more. Thanks!

Reply:
I don't agree that "gradient is correct". The backward method multiplies its argument with the gradient of the function, which is the VJP of course. I'd personally remove "from output to input", as this terminology only makes sense if you think about this is inserted as a layer in a feed-forward network.
```suggestion
* `torch.autograd.Function`: the `forward` method (forward pass) is the evaluation of the function, and `backward` method (backward pass) multiplies its argument with the gradient of the function, i.e. computes the Vector-Jacobian-adjoint-Product (VJP).
```